spec/test_collector/object_spec.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+require "timeout"
+
+RSpec.describe Buildkite::TestCollector::CI do
+  it "sleep without duration should not error" do
+    Buildkite::TestCollector::Object.configure
+
+    expect do
+      ::Timeout.timeout(0.5) do
+        sleep
+      end
+    end.to raise_error(Timeout::Error)
```

Comment:
without the fix, this will raise an `ArgumentError` when you call `sleep`:

```
  1) Buildkite::TestCollector::CI sleep without duration should not error
     Failure/Error:
       expect do
         ::Timeout.timeout(0.5) do
           sleep
         end
       end.to raise_error(Timeout::Error)

       expected Timeout::Error, got #<ArgumentError: wrong number of arguments (given 0, expected 1)> with backtrace:
         # ./lib/buildkite/test_collector/object.rb:6:in `sleep'
         # ./spec/test_collector/object_spec.rb:10:in `block (4 levels) in <top (required)>'
         # ./spec/test_collector/object_spec.rb:9:in `block (3 levels) in <top (required)>'
         # ./spec/test_collector/object_spec.rb:8:in `block (2 levels) in <top (required)>'
```